.circleci/config.yml

```
@@ -16,8 +16,9 @@ jobs:
       - run: &style-check
           name: Style check
           command: |
-            python -m pip install --progress-bar off --user -U flake8 flake8-black flake8-bugbear flake8-docstrings flake8-rst-docstrings pep8-naming flake8-isort
-            python -m flake8 --show-source ./
+            python -m pip install --progress-bar off --user -U pre_commit
```

The package name is `pre-commit`. It is correct to invoke it as `python -m pre_commit` if you don't want to use the alias `pre-commit`.
```suggestion
            python -m pip install --progress-bar off --user -U pre-commit
```

---

PyPI has a redirect for `pre_commit` so that you can use either safely.

---

...but let's use the canonical name. Those redirection rules could change in the future.

---

I mean sure I can change it, but I think that's _highly_ unlikely given how many packages I've seen using `pre_commit` in the past. And redirects aren't exactly hard to maintain.
Regarding using `pre-commit` vs `python -m pre_commit`, I find it preferable to sidestep potential issues with what's on the PYTHONPATH where possible. I could make this even cleaner and less error-prone by using a virtual environment, but I'm punting on that for now.